pkg/process/util/chunking.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package util
```

Comment:
WDYT about splitting up this PR into:
- moving/extracting the packages (get a buy-in from the respective teams like processes etc.)
- importing that package and implementing what is needed

That would make the review easier due to: 
- less teams are involved on the second PR
- less code to review